internal/sliceutils/sliceutils_test.go

```diff
@@ -82,3 +82,38 @@ func TestIntersection(t *testing.T) {
 		})
 	}
 }
+
+type intStruct struct {
+	i int
+}
+
+func TestMap(t *testing.T) {
+	t.Parallel()
+
```

Comment:
We can move this struct definition into the only function which uses it:

```suggestion
func TestMap(t *testing.T) {
	t.Parallel()

	type intStruct struct {
		i int
	}
```